trackersite/tracker/views.py

```diff
@@ -203,26 +245,34 @@ def mediainfo_formfield(f, **kwargs):
 mediainfoformset_factory = curry(inlineformset_factory, Ticket, MediaInfo,
     formset=ExtraItemFormSet, fields=MEDIAINFO_FIELDS, formfield_callback=mediainfo_formfield)
 
-EXPEDITURE_FIELDS = ('description', 'amount')
+EXPEDITURE_FIELDS = ('description', 'amount', 'wage')
 expeditureformset_factory = curry(inlineformset_factory, Ticket, Expediture,
     formset=ExtraItemFormSet, fields=EXPEDITURE_FIELDS)
 
+PREEXPEDITURE_FIELDS = ('description', 'amount', 'wage')
+preexpeditureformset_factory = curry(inlineformset_factory, Ticket, Preexpediture,
+    formset=ExtraItemFormSet, fields=PREEXPEDITURE_FIELDS)
+
 @login_required
 def create_ticket(request):
     MediaInfoFormSet = mediainfoformset_factory(extra=2, can_delete=False)
     ExpeditureFormSet = expeditureformset_factory(extra=2, can_delete=False)
-    
+    PreexpeditureFormSet = preexpeditureformset_factory(extra=2, can_delete=False)
+
     if request.method == 'POST':
         ticketform = TicketForm(request.POST)
         try:
             mediainfo = MediaInfoFormSet(request.POST, prefix='mediainfo')
             expeditures = ExpeditureFormSet(request.POST, prefix='expediture')
+            preexpeditures = PreexpeditureFormSet(request.POST, prefix='preexpediture')
             mediainfo.media   # trigger ValidationError when management form field are missing
             expeditures.media # this seems to be a regression between Django 1.3 and 1.6
+            preexpeditures.media # test
```

Comment:
test?

Reply:
Teď jen přijít na to, co jsem testoval. Dík za zachycení, zbytek je na mě :D.

Reply:
Jo, tohle je problém s nepopsaným kódem bez testů. A teď si představ že k tomu někdo přijde za dalších pět let :-)